lib/logging.js

```diff
@@ -140,4 +140,17 @@ function printUpgrades(options, { current, upgraded, numUpgraded, total, ownersC
   }
 }
 
-module.exports = { print, printJson, printUpgrades, toDependencyTable }
+function printIgnoredUpdates(options, ignoredUpdates) {
+  print(options, `\nIgnored incompatible updates (peer dependencies):\n`)
+  const table = createDependencyTable()
+  const rows = Object.entries(ignoredUpdates).map(([pkgName, { from, to, reason }]) => {
+    const strReason = 'reason: ' + Object.entries(reason)
+      .map(([pkgReason, requirement]) => pkgReason + ' requires ' + requirement)
+      .join(', ')
+    return [pkgName, from, '→', colorizeDiff(from, to), strReason]
+  })
+  rows.forEach(row => table.push(row)) // eslint-disable-line fp/no-mutating-methods
```

Comment:
Sorry for this, but I don't know how to do it in another way.

The same approach is already in the existing code:
https://github.com/raineorshine/npm-check-updates/blob/v11.5.3/lib/logging.js#L93

Reply:
Thanks for calling it out providing justification! No problem.

Actually, it annoyed me so I contributed a PR to their repo 😆
https://github.com/Automattic/cli-table/pull/139